recipes/macse/meta.yaml

```diff
@@ -1,29 +1,30 @@
 {% set name = "macse" %}
-{% set version = "1.2" %}
-{% set sha256hash = "5545afd948cc4cd8af435ad9b29c78f77226b73c505b4a42cf938aad0d256803" %}
+{% set version = "2.07" %}
+{% set sha256hash = "96873a1465f1e1aa9d0c462d469eee6954d36e8639727f69f34a72fcb6583963" %}
 
 package:
   name: {{ name }}
   version: {{ version }}
 
 source:
   # url: https://bioweb.supagro.inra.fr/{{ name }}/releases/{{ name }}_v{{ version }}.jar
-  url: https://bioweb.supagro.inra.fr/macse/releases/macse_v1.2.jar
+  # url: https://github.com/ranwez/MACSE_V2_PIPELINES/blob/master/UTILS/macse_v2.07.jar?raw=true
+  url: https://bioweb.supagro.inra.fr/macse/releases/macse_v2.07.jar
```

Comment:
```suggestion
    url: https://bioweb.supagro.inra.fr/macse/releases/macse_v2.07.jar
```

Reply:
Apparently the problem was related to the url jar path. I would have prefer to get the jar from github 
but replacing  url: https://github.com/ranwez/MACSE_V2_PIPELINES/blob/master/UTILS/macse_v2.07.jar?raw=true
by
 url: https://bioweb.supagro.inra.fr/macse/releases/macse_v2.07.jar
fix problems ....

Reply:
I have commited the suggested modification (cleaning yaml file). So everything should be fine now.

Reply:
thanks again for your help